test/integration/features/generic-driver.feature

```diff
@@ -0,0 +1,60 @@
+@generic-driver
+Feature: With generic driver Minishift can provision remote unprovisioned VM 
+
+  Scenario: User creates an unprovisioned VM
+    Given setting up environment variable "MINISHIFT_ENABLE_EXPERIMENTAL" with value "y" succeeds
+     When executing "minishift profile set remote" succeeds
+      And executing "minishift start --no-provision" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: Provisioning remote instance
+    Given unset environment variable "MINISHIFT_ENABLE_EXPERIMENTAL" succeeds
+      And setting scenario variable "REMOTE_VM_IP" to the stdout from executing "minishift ip"
+      And executing "minishift profile set minishift" succeeds
```

Comment:
Add ```And image caching is disabled``` before m start

Reply:
@amitkrout fixed, thanks, I've missed that :)